modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java

```diff
@@ -1904,10 +1904,18 @@ private void _notifySubscribers(
 		boolean sendEmailEntryUpdated = GetterUtil.getBoolean(
 			serviceContext.getAttribute("sendEmailEntryUpdated"));
 
+		Map<String, Serializable> attributes = serviceContext.getAttributes();
+
+		boolean updateTriggeredByScheduler = !attributes.containsKey(
+			"sendEmailEntryUpdated");
+
 		if (serviceContext.isCommandAdd() &&
 			blogsGroupServiceSettings.isEmailEntryAddedEnabled()) {
 		}
-		else if (sendEmailEntryUpdated && serviceContext.isCommandUpdate() &&
+		else if ((sendEmailEntryUpdated ||
```

Comment:
Maybe a reorder of this condition will make things fail faster if you put the `serviceContext.isCommandUpdate() &&  blogsGroupServiceSettings.isEmailEntryUpdatedEnabled()` before the new expression that makes more operations